pixart.js

```diff
@@ -1 +1,27 @@
 
+$("#set-color").on("click", function(){
+  event.preventDefault();
+  $(".brush").css("backgroundColor", $("#color-field").val());
+
+  //should be a way of abstracting $("#color-field").val() into a variable, since it repeats three times.
+})
+
+$("#set-color").on("keypress", function(){
+  event.preventDefault();
+  $(".brush").css("backgroundColor", $("#color-field").val());
+})
```

Comment:
If you change one of these event listeners to listen for the "submit" event instead of ether "click" or "keypress" you will only need to have one of these listeners.